Assets/UI/Popups/techciviccompletedpopup.lua

```diff
@@ -406,6 +406,8 @@ function OnInit( isReload:boolean )
     if isReload then
         LuaEvents.GameDebug_GetValues(RELOAD_CACHE_ID);
     end
+
+    LateInitialize();
```

Comment:
matching the unmodified code, they added in an empty LateInitialize to override stuff